contracts/tokenomics/maker/src/utils.rs

```diff
@@ -130,3 +138,56 @@ pub fn validate_bridge(
 
     Ok(bridge_pool)
 }
+
+/// # Description
+/// Checks if from_token - UST pool exists
+/// # Params
+/// * **deps** is the object of type [`DepsMut`].
+///
+/// * **cfg** is the object of type [`Config`].
+///
+/// * **from_token** is the object of type [`AssetInfo`].
+pub fn uusd_pool_exists(deps: Deps, cfg: &Config, from_token: AssetInfo) -> Option<PairInfo> {
```

Comment:
I think function name *_exists implies to return `bool` type. I suggest to rename it to `try_get_uusd_pool`